python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -612,6 +614,16 @@ def uniform_neighbor_sample(
         fanout_vals = fanout_vals.get().astype("int32")
     elif isinstance(fanout_vals, cudf.Series):
         fanout_vals = fanout_vals.values_host.astype("int32")
+    elif isinstance(fanout_vals, dict):
+        # FIXME: Add expensive check to ensure all dict values are lists
+        # Convert to a tuple of sequence (edge type size and fanout values)
+        edge_type_size = []
+        [edge_type_size.append(len(s)) for s in list(fanout_vals.values())]
+        edge_type_fanout_vals = list(chain.from_iterable(list(fanout_vals.values())))
+        fanout_vals = (
```

Comment:
Per my earlier suggestions, I think we want this to be a CSR structure, so converting from a list of sizes to a list of offsets is perhaps best done here.

Reply:
We changed this back to a dense structure... so I think this code isn't right.

Reply:
This still seems wrong to me.  If you want to support fanout_vals as a dictionary I think we need to convert it to a dense array to get the right values.  Do you have a python test for this path that we can verify?